packages/widgets-manager/src/__tests__/grid.spec.ts

```diff
@@ -0,0 +1,184 @@
+import { GridManager } from '../grid';
+import { AppendAppStrategy } from '../strategies/mounting/append';
+import { App } from '../types';
+const windowObject: Window = window;
+
+const mockApp1: App = {
+  id: 1,
+  name: 'Test App',
+  payload_type: 'html',
+  payload: '',
+  settings: {
+    css: '',
+    inlineCss: {},
+    position: {} as any,
+  },
+  organization_id: 1,
+}
+
+const mockApp2: App = {
+  id: 2,
+  name: 'Test App2',
+  payload_type: 'html',
+  payload: '',
+  settings: {
+    css: '',
+    inlineCss: {},
+    position: {} as any,
+  },
+  organization_id: 1,
+}
+
+describe('GridManager tests', () => {
```

Comment:
What exactly are you describing with these tests? you certainly aren't describing tests with your tests